diff -aur ../build/extbld/third_party/bsp/stmf7cube/cube/STM32CubeF7-1.17.1/Drivers/BSP/Components/wm8994/wm8994.c STM32CubeF7-1.17.1/Drivers/BSP/Components/wm8994/wm8994.c
--- STM32CubeF7-1.17.1/Drivers/BSP/Components/wm8994/wm8994.c
+++ ../build/extbld/third_party/bsp/stmf7cube/cube/STM32CubeF7-1.17.1/Drivers/BSP/Components/wm8994/wm8994.c
@@ -122,6 +122,35 @@ static uint8_t CODEC_IO_Write(uint8_t Addr, uint16_t Reg, uint16_t Value);
 /** @defgroup WM8994_Private_Functions
   * @{
   */ 
+typedef struct
+{
+  uint16_t enable:1;
+  uint16_t gain3:5;
+  uint16_t gain2:5;
+  uint16_t gain1:5;
+} stReg;
+
+typedef union
+{
+  uint16_t u16;
+  stReg st;
+} uRegData;
+
+void setEq(uint16_t DeviceAddr)
+{
+  uRegData data;
+
+  data.st.enable = 1;
+  data.st.gain1 = 0xc;
+  data.st.gain2 = 0xc;
+  data.st.gain3 = 0x0;
+  CODEC_IO_Write(DeviceAddr, 0x480, data.u16);
+  data.st.enable = 0;
+  data.st.gain1 = 0x0;
+  data.st.gain2 = 0x0;
+  data.st.gain3 = 0x0;
+  CODEC_IO_Write(DeviceAddr, 0x481, data.u16);
+}
 
 /**
   * @brief Initializes the audio codec and the control interface.
@@ -269,6 +298,8 @@ uint32_t wm8994_Init(uint16_t DeviceAddr, uint16_t OutputInputDevice, uint8_t Vo
       counter += CODEC_IO_Write(DeviceAddr, 0x605, 0x0000);
       break;
     }
+
+    setEq(DeviceAddr);
   }
   else
   {
